### DISCUSSION TITLE
Is there possiblities to know how much gift card value applied to checkout?

### DISCUSSION THREAD

**Discussion by siddharth222** [Shopify/function-examples#253] [Feedback: Payment Customization API]
i want hide a payment method based on the final total. Ex: hide COD order total over 160.
According to my example, it hides the COD as Function Extention returns 200 but the checkout total is 150.

![image](https://github.com/Shopify/function-examples/assets/66834661/89554849-f189-4914-b37f-629d5226f7c9)

![image](https://github.com/Shopify/function-examples/assets/66834661/cfc9bda9-114a-4e27-b71f-a2366d99bf0d)

**Comment by AndreaMorone** [comment]
We are interested too. Bump for this functionality ⬆️
